src/basic/siphash24.h

```diff
@@ -1,3 +1,5 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
```

Comment:
hmm, more correct is probably "CC0 Public Domain Dedication", seep siphash24.c

Reply:
Indeed, I missed that the C file specifies CC0. I made it  LGPL-2.1+ because our header is mostly independent of the C file, and I thought it's better to assign *some* specific license. But yeah, it's better to make both CC0.